resteasy/src/main/java/com/cerner/beadledom/resteasy/ResteasyContextListener.java

```diff
@@ -76,4 +70,21 @@ protected void withInjector(Injector injector) {
    * Returns the list of Guice modules to be used with Resteasy.
    */
   protected abstract List<? extends Module> getModules(ServletContext context);
+
+  /**
+   * Processes module bindings in the Guice injector.
+   */
+  @VisibleForTesting
+  public void processInjector(ServletContext context, Injector injector) {
```

Comment:
somehow I am not comfortable having this method. Since this is the initial point of a Web application can the Injector that is created in this class even have a parent Injector ? What if we remove this loop completely  and just have the processor process the injector that is created in this class ?